stubs/2.7/traceback.pyi

```diff
@@ -0,0 +1,16 @@
+from typing import Any, IO, AnyStr, Callable, Tuple, List
+from types import TracebackType, FrameType
+
+def print_tb(traceback: TracebackType, limit: int = None, file: IO[AnyStr] = None) -> None: ...
+def print_exception(type: type, value: Exception, limit: int = None, file: IO[AnyStr] = None) -> None: ...
+def print_exc(limit: int = None, file: IO[AnyStr] = None) -> None: ...
+def format_exc(limit: int = None) -> None: ...
+def print_last(limit: int = None, file: IO[AnyStr] = None) -> None: ...
+def print_stack(f: FrameType, limit: int = None, file: IO[AnyStr] = None) -> None: ...
+def extract_tb(f: TracebackType, limit: int = None) -> List[Tuple[AnyStr, int, AnyStr, AnyStr]]: ...
```

Comment:
You should not have `AnyStr` only in the return type (mypy should complain about it) since it's a type variable and we can't generally infer a value if no argument uses it. Probably use `str` here.